specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -865,14 +1315,14 @@
           "EnrollmentAccounts"
         ],
         "x-ms-examples": {
-          "EnrollmentAccountsListByBillingAccountName": {
-            "$ref": "./examples/EnrollmentAccountsListByBillingAccountName.json"
+          "EnrollmentAccountsListByBillingAccount": {
+            "$ref": "./examples/EnrollmentAccountsListByBillingAccount.json"
           },
-          "EnrollmentAccountsListByBillingAccountNameWithExpand": {
-            "$ref": "./examples/EnrollmentAccountsListByBillingAccountNameWithExpand.json"
+          "EnrollmentAccountsListByBillingAccountWithExpand": {
+            "$ref": "./examples/EnrollmentAccountsListByBillingAccountWithExpand.json"
           }
         },
-        "operationId": "EnrollmentAccounts_ListByBillingAccountName",
+        "operationId": "EnrollmentAccounts_ListByBillingAccount",
```

Comment:
operationId  change will cause  SDK breaking change

Reply:
Undone the change for this specific API.